tests/tests.go

```diff
@@ -25,6 +25,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/rand"
 	"k8s.io/client-go/util/retry"
 	"k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
```

Comment:
Please note the existing "k8s.io/utils/pointer" module provide the same functionality as "k8s.io/utils/pointer", adding the latter adds a lot of noise to the PR.

Adding ptr module w/o removing the pointer one make the codebase inconsistent,
The next dev may get confuse, arbitrarily pick one of them and make the code inconsistency worse.

If you intend to replace one module with another, I think it will be better doing that on a dedicated PR.

Reply:
> Please note the existing "k8s.io/utils/pointer" module provide the same functionality as "k8s.io/utils/pointer",

existing "k8s.io/utils/pointer" is about converting simple data to standard params (int64, etc..)
"k8s.io/utils/ptr" is different entirely and is about converting values to pointers (To).
AFAIK They don't share function names, so there shouldn't be a problem. 

Can you elaborate?

Reply:
"k8s.io/utils/pointer" enable converting types to pointers, for example `pointer.Bool(myboolean), ponter.String(mystring)` .
 "k8s.io/utils/ptr" enable the same functionality implemented using generics for example: `ptr.To(mystring)` returns `*string`
In practice "k8s.io/utils/pointer"  uses  "k8s.io/utils/ptr".

See this example for replacing one with the other in k8s/k8s https://github.com/kubernetes/kubernetes/issues/130408 

WDYT?

Reply:
ah.. then I'm all forward on doing it, but on a follow-up PR